scaffold/github/workflows/Security.yml

```diff
@@ -0,0 +1,86 @@
+name: "Security"
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  Security:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Checkout base branch
+        run: |
+          git fetch --no-tags --depth=1 origin ${{ github.base_ref }}
+          git checkout ${{ github.base_ref }}
+          git checkout -
+
+      - uses: actions/cache@v4
+        with:
+          path: ${{ github.workspace }}/.ddev/.drainpipe-composer-cache
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - uses: ./.github/actions/drainpipe/set-env
+
+      - name: Install and Start DDEV
+        uses: ./.github/actions/drainpipe/ddev
+        with:
+          git-name: Drainpipe Bot
+          git-email: no-reply@example.com
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+          ssh-known-hosts: ${{ secrets.SSH_KNOWN_HOSTS }}
+
+      - name: Build Project
+        run: ddev composer install
+
+      - name: Run Security Checks
+        run: ddev task security composer_lock_diff_from=${{ github.base_ref }} composer_lock_diff=composer-lock-diff.md
+
+      - name: Publish Results
+        run: |
+          COMPOSER_LOCK_DIFF="<!-- composer-lock-diff -->
```

Comment:
Can we make this at least a shell command or a task, so we can run and test this outside of GitHub actions?

Reply:
This part is all very GitHub actions specific and relies on variables set in the environment, so doesn't really make sense to be in a task